src/PatientChecking/PatientChecking/Views/Patient/Detail.cshtml

```diff
@@ -308,106 +323,106 @@
         $(".patient-box-detail :input").prop("disabled", true);
     });
 
-    $("#search-input").focus(function () {
-        $(".search-result-container").css("display", "flex");
+    searchInput.focus(function () {
+        searchResultContainer.css("display", "flex");
     });
 
     $(".search-result-container ul li a").click(function () {
-        $(".search-result-container").css("display", "none");
+        searchResultContainer.css("display", "none");
     });
 
-    $("#search-input").change(function () {
+    searchInput.change(function () {
         var query = $(this).val();
     });
 
-    $("#btn-edit").click(function () {
-        $("#btn-edit").css("display", "none");
-        $(".edit-mode").css("display", "flex");
+    btnEdit.click(function () {
+        btnEdit.css("display", "none");
+        editMode.css("display", "flex");
         $(".patient-box-detail :input").prop("disabled", false);
         $("#fullNameInput").prop("disabled", true);
 
         if (nationality !== "Vietnamese") {
-            $("#ethnicSelect").prop("disabled", true);
+            ethnicSelect.prop("disabled", true);
         }
 
-        var idNumber = $("#idCardNo").val();
+        var idNumber = idCardNo.val();
 
         if (idNumber === "") {
-            $("#issuedDate").prop("disabled", true);
-            $("#issuedPlace").prop("disabled", true);
+            issuedDate.prop("disabled", true);
+            issuedPlace.prop("disabled", true);
         }
     });
 
     $("#btn-cancel").click(function () {
```

Comment:
? please correct this one as well

Reply:
I have applied this as per the comment above. Please review this @minhphien